docs-src/references/web-ui-server-env-vars.md

```diff
@@ -0,0 +1,249 @@
+---
+id: web-ui-server-env-vars
+title: Temporal Web UI environmental variables
+sidebar_label: Web UI environmental variables
+description: How to set  environmental variables for Temporal Web UI.
+tags:
+  - docker
+  - webui
+  - ui server
+---
+
+Environmental variables are fields that are set to configure a Temporal Cluster.
+This allows the user to configure the Cluster in any environment, such as a Docker container.
+
+Use `docker run` as the image name needed to configure the Web UI environmental variables.
+`docker run` requires at least 1 argument.
+
+`docker run [options] <image> <command> [args]`
```

Comment:
Not sure we want to document docker commands here.
An example of using the env vars is what we are after - 
I suggest finding the Docker CLI docs - and link to those instead.